docs/howto/danceable-lyrics-input-tool/README.md

```diff
@@ -0,0 +1,485 @@
+
+```bash
+~/MyWorkspace$ ayx_plugin_cli create-yxi
+[Creating YXI] started
+[Creating YXI] -- generate_config_files:generate_config_xml
+[Creating YXI] -- generate_config_files:generate_tool_config_xml
+[Creating YXI] .  generate_config_files:generate_manifest_jsons
+[Creating YXI] Generating manifest.json for DanceableLyrics...
+[Creating YXI] Done!
+...omitted...
+~\MyWorkspace\main.pyz -e ayx_python_sdk.providers.amp_provider.__main__:main
+[Creating YXI] Created shiv artifact at: ~\MyWorkspace\main.pyz
+[Creating YXI] .  create_yxi:create_yxi
+[Creating YXI] finished
+```
+
+# Run the Test Client
+Before proceeding on to running within Designer, it's good practice to do some basic testing with `ayx-sdk-cli`'s `plugin run` command to check for any errors. From your workspace:
+
+```bash
+ayx-sdk-cli plugin run DanceableLyrics -o danceable.csv
+```
+
+The above command will run our plugin and produce `danceable.csv`. You may run `ayx-sdk-cli plugin run --help` for additional options.
+
+Given the command above, our output should look something like this (I am using [Rich CLI](https://github.com/Textualize/rich-cli) to preview):
+
+![danceable.csv](./assets/danceable-csv.png)
+
+# Install and Run in Designer
+In this section we will go over the two ways to install the plugin into Designer.
+
+## Method 1
+After you create a .yxi, you can double-click the .yxi to install it into Designer. This opens Designer and prompts you to install the package in a new dialog box. It looks something like this:
+
+![Designer Install Prompt](./assets/designer-install-prompt.png)
+
+Once it installs, you can find the plugin under the `Python SDK Examples` tool tab.[^2]
+
+## Method 2
+You can also create the `.yxi` _**and**_ install it all at once via the `ayx_plugin_cli designer-install` command. Choose the install option that matches your Designer install. Typically, this is the `user` install option.
+
+```bash
+~/MyWorkspace$ ayx_plugin_cli designer-install
+Install Type (user, admin) [user]: user
+[Creating YXI] started
+[Creating YXI] -- generate_config_files:generate_config_xml
+[Creating YXI] -- generate_config_files:generate_tool_config_xml
+[Creating YXI] .  generate_config_files:generate_manifest_jsons
+[Creating YXI] Generating manifest.json for DanceableLyrics...
+[Creating YXI] Done!
+...omitted...
+[Creating YXI] finished
+[Installing yxi ~\MyWorkspace\build\yxi\DanceableLyrics.yxi into designer] started
+[Installing yxi ~\MyWorkspace\build\yxi\DanceableLyrics.yxi into designer] .  install_yxi
+[Installing yxi ~\MyWorkspace\build\yxi\DanceableLyrics.yxi into designer] finished
+If this is your first time installing these tools, or you have made modifications to your ayx_workspace.json file, please restart Designer for these changes to take effect. # <-- Done installing into Designer!
+```
+
+Once the command finishes you can open Designer and find your tool under the `Python SDK Examples` tab.[^1]
```

Comment:
TW review:

Once the command finishes, you can open Designer and find your tool under the `Python SDK Examples` tool category.[^1]